apps/marketing/src/components/(marketing)/widget.tsx

```diff
@@ -90,10 +91,10 @@ export const Widget = ({ className, children, ...props }: WidgetProps) => {
     }
```

Comment:
I'm curious about the reason for using `useMemo` here. Do you think it's necessary?